lorax.spec

```diff
@@ -224,10 +224,10 @@ getent passwd weldr >/dev/null 2>&1 || useradd -r -g weldr -d / -s /sbin/nologin
 %dir %attr(0771, root, weldr) %{_sharedstatedir}/lorax/composer/blueprints/
 %attr(0771, weldr, weldr) %{_sharedstatedir}/lorax/composer/blueprints/*
 
-%files -n composer-cli
```

Comment:
This seems to be renaming the subpackage. I don't think that's what you want. If you do want that, you should probably open a separate pull request, as it involves different work.